checkins.json

```diff
@@ -1,47 +1,82 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "description": "collection of check-ins",
+  "description": "A collection of check-ins",
   "type": "object",
   "properties": {
-    "checkins": {
-      "description": "an array of check-ins",
-      "id": "checkins",
+    "checkinItems": {
+      "description": "List of checkin",
+      "id": "checkinItems",
       "type": "array",
-      "items": {
+      "checkinItems": {
         "type": "object",
         "properties": {
           "po_line_id": {
-            "description": "UUID of po line this check-in is related to",
+            "description": "The id of the checkin po_line",
             "type": "string",
             "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
           },
-          "received": {
-            "description": "number of pieces received",
+          "numberOfItemsCheckedin": {
+            "description": "The number of items to check-in",
             "type": "integer"
           },
           "pieces": {
-            "description": "an array of pieces details",
+            "description": "A collection of piece details",
+            "id": "pieces",
             "type": "array",
-            "items": {
+            "piece": {
```

Comment:
Ideally we'd just reference the piece schema from https://github.com/folio-org/acq-models/pull/55

Reply:
@craigmcnally The piece schema put together in MODORDSTOR-30 has lesser fields than in MODORDERS-90. Are the other fields(ex: electronic_book_plate etc) are part of checkin and not pieces?